src/main/scala/firrtl/passes/ExpandWhens.scala

```diff
@@ -103,12 +103,16 @@ object ExpandWhens extends Pass {
                       defaults: Defaults,
                       p: Expression)
                       (s: Statement): Statement = s match {
+        case stmt @ (_: DefNode | EmptyStmt) => stmt
         case w: DefWire =>
           netlist ++= (getFemaleRefs(w.name, w.tpe, BIGENDER) map (ref => we(ref) -> WVoid))
           w
         case w: DefMemory =>
           netlist ++= (getFemaleRefs(w.name, MemPortUtils.memType(w), MALE) map (ref => we(ref) -> WVoid))
           w
+        case w: WDefInstance =>
+          netlist ++= (getFemaleRefs(w.name, w.tpe, MALE).map(ref => we(ref) -> WVoid))
```

Comment:
`MALE` of `getFemaleRefs` here means to get all the inputs to the submodule?

Reply:
Clarified offline

Reply:
That's correct, similar to how it's used on the `DefMemory` above.